app/components/post_header/post_header.js

```diff
@@ -95,6 +98,19 @@ export default class PostHeader extends PureComponent {
                     </View>
                 </TouchableOpacity>
             );
+        } else if (isGuest) {
```

Comment:
so a guest in this case is a third class citizen? if that is the case good, if not then change the order

Reply:
Yes, if is something that comes from a webhook it will show to `bot` badge, if it is a bot it will show the `bot` badge too, and only if is not a bot or from a webhook , if the user is a guest it will show the `guest` badge, I think makes sense, because a bot is a bot (is really rare to have a guest bot, and anyway is a bot with restricted permissions, but a bot)